### ISSUE TITLE
version 5.22.6 import Finch change

### ISSUE THREAD

**Issue by Cam-Bread** [Finch-API/finch-api-node#360]
It looks like in the latest version of @tryfinch/finch-api, (5.22.6) the default Finch export was removed.
This means your readme instructions are incorrect: 
```typescript
import Finch from '@tryfinch/finch-api';

const finch = new Finch({
  accessToken: 'My Access Token',
});
```
this - any code written following it - will now trigger an error:
```error
Type error: This expression is not constructable.
12:01:40.416 | Type 'typeof import("/vercel/path0/node_modules/.pnpm/@tryfinch+finch-api@5.22.6/node_modules/@tryfinch/finch-api/index")' has no construct signatures.
```
In my code I had to update the import as follows to resolve the error:
```typescript
import {Finch} from "@tryfinch/finch-api";
```
easy fix but I am sure it will confuse people just setting things up if the readme instructions are incorrect.

**Comment by RobertCraigie** [comment]
This has been fixed in `5.22.7`

**Comment by Cam-Bread** [comment]
That was quick! thanks :)
